server/src/main/java/io/crate/execution/dsl/projection/ColumnIndexWriterProjection.java

```diff
@@ -229,6 +248,12 @@ public int hashCode() {
     public void writeTo(StreamOutput out) throws IOException {
         super.writeTo(out);
 
+        if (out.getVersion().onOrAfter(Version.fromString("5.3.4"))) {
+            out.writeVInt(partitionedByColumns.size());
```

Comment:
Maybe `.after(Version.5_3_3)` would be better in order not to use constant instead of `String`?